backend/tests/Feature/UserAuthTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Entities\User;
+use Tests\TestCase;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+
+class UserAuthTest extends TestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+        User::where('email','test@gmail.com')->delete();
```

Comment:
instead of this use trait RefreshDatabase https://laravel.com/docs/5.8/database-testing#resetting-the-database-after-each-test

and factory(User::class)->create()

Reply:
and create user in test